EvenSum.kt

```diff
@@ -0,0 +1,20 @@
+class EvenSum {
+    fun evenSum(array: Array<Int>) : Int
+    {
+        var i:Int = 0
+        var sum:Int = 0
+        for(i in 0..array.size-1 step 1)
+        {
+            println(array[i])
+        }

```

Comment:
Not sure why printing helps here, it was not in the req of the problem.

If one does want to keep printing please consider merging the 2 loops.

Reply:
You are right, did it for trying things, and  I should've removed it as it was not required.